_server.js

```diff
@@ -0,0 +1,158 @@
+const electron = require('electron'),
+app = electron.app,
+BrowserWindow = electron.BrowserWindow,
+ipcMain = electron.ipcMain
```

Comment:
Update: See commit 14fdef4 for linted JavaScript files. Since running the formatter after fixing lint errors results in more lint errors, I decided to run the formatter first and linter second.